code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -7,7 +7,10 @@
 	return 0
 
 /mob/living/carbon/human/canBeHandcuffed()
-	return 1
+	if(get_num_arms() >= 2)
+		return 1
+	else
+		return 0

```

Comment:
No way would you use true/false here. Those are used in a yes/no context, not a "number of an object" context